package.json

```diff
@@ -4,13 +4,16 @@
   "description": "LoopBack SOAP Web Services Connector",
   "main": "index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha -R spec --timeout 30000 test/*test.js"
+    "test": "./node_modules/.bin/mocha -R spec --timeout 30000 test/*test.js",
+    "pretest": "babel node_modules/strong-soap/src -d node_modules/strong-soap/lib"
```

Comment:
@raymondfeng , doing an npm install, actually brings the lib folder. it is only not working in the CI test environment